libs/librepcb/core/project/board/drc/boarddesignrulecheckdata.h

```diff
@@ -100,6 +104,7 @@ struct BoardDesignRuleCheckData final {
     UnsignedLength minWidth;
     Path outline;
     QVector<Path> fragments;
+    NetSignal* netSignal;  // May be null.
```

Comment:
Same here - the existing member `net` should be used instead.

And don't forget to also remove the `#include` for `netsignal.h` :nerd_face: